hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/MasterStatusTmpl.jamon

```
@@ -652,7 +652,10 @@ AssignmentManager assignmentManager = master.getAssignmentManager();
     <%for TableDescriptor desc : tables%>
     <%java>
       TableName tableName = desc.getTableName();
-      TableState tableState = master.getTableStateManager().getTableState(tableName);
+      TableState tableState = master.getTableStateManager().getTableState(tableName, true);
```

We can just catch the exception here?

---

Yes, we can just catch the exception here.. but I have added a separate method that can ignore throwing exception which could be used in other scenarios also where we want to ignore this exception. one such scenario is mentioned in below JIRA
https://issues.apache.org/jira/browse/HBASE-28159